webpack/assets/javascripts/react_app/components/notifications/toast/Toast.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Icon from '../../common/Icon';
+import helpers from '../../../common/helpers';
+import { ALERT_CSS } from '../../../constants';
+import CloseButton from '../../common/CloseButton';
+import * as ToastActions from '../../../redux/actions/toasts';
+// eslint-disable-next-line no-unused-vars
+import Timer from '../../common/Timer';
+
+const timerDelay = 8000;
+
+class Toast extends React.Component {
+  constructor(props) {
+    super(props);
+    helpers.bindMethods(this, ['dismiss', 'execute', 'onMouseEnter', 'onMouseLeave']);
+  }
+
+  componentDidMount() {
+    if (!this.props.sticky) {
+      this.timer = new Timer(this.execute, timerDelay);
+      this.timer.startTimer();
+    }
+  }
+
+  componentWillUnmount() {
+    if (this.timer) {
+      this.timer.clearTimer();
+      this.timer = null;
+    }
+  }
+
+  dismiss() {
+    ToastActions.hideToast(this.props.id);
+  }
+
+  execute() {
+    if (this.timer) {
+      this.timer.clearTimer();
+      this.timer = null;
+    }
+    this.dismiss();
+  }
+
+  onMouseEnter() {
+    if (this.timer) {
+      this.timer.clearTimer();
+    }
+  }
+
+  onMouseLeave() {
+    if (this.timer) {
+      this.timer.startTimer();
+    }
+  }
+
+  render() {
+    const { type, title, dismissable, link, message } = this.props;
+
+    return (
+      <div className={ALERT_CSS[type] + ' toast-pf'}
+           onMouseEnter={this.onMouseEnter} onMouseLeave={this.onMouseLeave} >
+        {dismissable && <CloseButton onClick={this.dismiss}></CloseButton>}
+
+        <div className="pull-right toast-pf-action">
+          <a href="#">
+            {link}
+          </a>
+        </div>
+        <Icon type={type}/>
+        <strong>
```

Comment:
It should be an inline element. You can see some examples in the patternfly reference markup